index.js

```diff
@@ -55,7 +59,9 @@ function parseRubyEngineAndVersion(rubyVersion) {
   } else { // engine-X.Y.Z
     [engine, version] = rubyVersion.split('-', 2)
   }
-
+  if (os.platform() === 'win32' && version === 'mingw') {
+    version = 'head'
+  }
```

Comment:
I think a nicer way to do this would be to simply have both `head` and `mingw` in windows-versions.js (with the same URL).

Reply:
Ok.  I wasn't sure what your thoughts about naming were.

I thought it would be helpful to have mingw master with 'ruby-head', 'head', and 'mingw'.  I'll remove the logic and add duplicate rows in windows-versions.js